cmd/ocm-backplane/login/login.go

```diff
@@ -152,6 +167,12 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 
 	targetCluster.Server = proxyURL
 
+	// Add proxy URL to target cluster
+	proxyURL, hasProxyUrl := os.LookupEnv(info.BACKPLANE_PROXY_ENV_NAME)
+	if hasProxyUrl {
+		targetCluster.ProxyURL = proxyURL
+	}
+
```

Comment:
what if the proxy is set in the config file, but not specified using --proxy ?
it looks like we should do like this:
1. If --proxy is used, use this proxy url and write it to kubeconfig
2. We should ignore that? since after login, you're already using this proxy when you run oc command
3. If the config already set this, read this proxy url and write to kubeconfig when login.

Reply:
If --proxy is not used, then proxy derived based on the following order 
1. via env vars 
https://github.com/openshift/backplane-cli/blob/67e50c71be30aad3c328a63cb3972ea2438bb9fc/pkg/utils/ocmWrapper.go#L237
2. `backplane.json` file 
https://github.com/openshift/backplane-cli/blob/67e50c71be30aad3c328a63cb3972ea2438bb9fc/pkg/utils/ocmWrapper.go#L245

I always write `proxy` into .kubeconfig if it's not empty. 

Am I missing something?

Reply:
As we discussed, This case is handled vis `http.transport`